lib/src/usage.dart

```diff
@@ -250,14 +307,19 @@ class Usage {
   }
 }
 
-/// Pads [source] to [length] by adding spaces at the end.
-String padRight(String source, int length) {
-  final result = new StringBuffer();
-  result.write(source);
-
-  while (result.length < length) {
-    result.write(' ');
-  }
-
-  return result.toString();
+bool isWhitespace(String text, int index) {
+  final int rune = text.codeUnitAt(index);
+  return rune >= 0x0009 && rune <= 0x000D ||
+      rune == 0x0020 ||
+      rune == 0x0085 ||
+      rune == 0x00A0 ||
+      rune == 0x1680 ||
+      rune == 0x180E ||
+      rune >= 0x2000 && rune <= 0x200A ||
+      rune == 0x2028 ||
+      rune == 0x2029 ||
+      rune == 0x202F ||
+      rune == 0x205F ||
+      rune == 0x3000 ||
+      rune == 0xFEFF;
```

Comment:
This would be a lot more readable if you used the charcode package. Also, I'm not sure this fancy Unicode whitespace logic actually provides much value.

At the very least, you shouldn't break on a non-breaking space (U+00A0).

Reply:
Fixed the non-breaking space thing. I agree it's probably overkill, but it's harmless too.

Reply:
I put it in because I just wanted it to work better than plain ASCII spaces with non-western locales, but yes, it's probably overkill.